modules/serverless-install-serving-web-console.adoc

```diff
@@ -51,4 +61,50 @@ image::serving-conditions-true.png[Conditions]
 ====
 It may take a few seconds for the Knative Serving resources to be created. You can check their status in the *Resources* tab.
 ====
-. If the conditions have a status of *Unknown* or *False*, wait a few moments and then check again after you have confirmed that the resources have been created.
++
+If the conditions have a status of *Unknown* or *False*, wait a few moments and then check again after you have confirmed that the resources have been created.
+
+. Check that the Knative Serving resources have been created:
++
+[source,terminal]
+----
+$ oc get pods -n knative-serving
+----
```

Comment:
This is in the web console docs 🤔  Should we be looking at terminal commands here?

Reply:
I'll remove it, wasn't sure if we wanted to give folks the option of checking this whatever way they installed it but that's OK

Reply:
I mean, it's totally fine. I was mostly wondering what the "policy" is in OCP docs wrt. using UI and Terminal in combo.

Reply:
It's a good question 😄 I'll find out...

Reply:
Decided to remove after talking to other docs folks.